examples/with-ai-sdk/app/MyRuntimeProvider.tsx

```
@@ -0,0 +1,68 @@
+import { assistantUIToolBox } from "@assistant-ui/react/runtimes/core/ThreadRuntimeCore";
+
+export const toolbox = assistantUIToolBox<Tools>();
+
+const GetLocationFromUser = toolbox.getLocationFromUser.getTool({
+  render: (a) => {
+    return (
+      <button
+        onClick={() => {
+          a.addResult("Mumbai");
+        }}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)